src/analytics/interface.js

```diff
@@ -29,9 +29,7 @@ import PropTypes from 'prop-types';
 
 const optionsShape = {
   config: PropTypes.object.isRequired,
-  httpClient: PropTypes.shape({
-    post: PropTypes.func.isRequired,
-  }).isRequired,
+  httpClient: PropTypes.func.isRequired,
```

Comment:
Somewhat confuses me, but turns out this is a `func` not an object.  Not sure why I didn't see a warning earlier...

Reply:
We had a test that mocked it as an object, which I think is where this came from.